tools/mo/setup.py

```diff
@@ -31,7 +31,7 @@ def read_text(path):
 requirements_txt = []
 py_modules = []
 for item in os.listdir():
-    if re.match(r'requirements(.*)\.txt', item):
+    if re.match(r'requirements_?(tf|tf2|onnx|mxnet|kaldi|caffe)?\.txt', item):
```

Comment:
I disagree with this change because we should always update it once there are new requirement files, as example,  for pytorch and paddle. Let us not change it.

Reply:
I understand what you mean and agree that probably it could be easier with `(.*)` but anyways if we add new requirements we still will need to add them separately below in the same `setup.py` 

https://github.com/openvinotoolkit/openvino/blob/4dd75a1b8be4351698a5f8a488a53eacd85a4ba4/tools/mo/setup.py#L107-L112

And one more factor. The reason i specified them manually is that `(.*)` copied also `requirements_dev.txt` which copies file with packages (pytest, generate and so on) that are not necessary when openvino-dev is installed from pip/wheel. Copying `requirements-dev.txt` didn't cause significant damage but still was not quite right previously.

Therefore i insist to keep this regexp pattern with manual setting of tf/tf_2/onnx/and so on

Reply:
Will it work for `requirement.txt` case?

Please fix the comment above about `CustomLayersMapping.xml*` because now we are adding new file into the package but it is not clear for what.

Reply:
Indeed `requirements.txt` is also copied by this regex. And it's right to be copied because it's used subsequently 
https://github.com/openvinotoolkit/openvino/blob/940844e71f6fa5603576b2d0d51da043a5634b0a/tools/mo/openvino/tools/mo/utils/versions_checker.py#L247-L258